### ISSUE TITLE
Chromatic: "Failed to extract stories from your Storybook" 

### ISSUE THREAD

**Issue by TomRadford** [stock2shop/console_v2#295]
Part of SvelteKit transition/Modernisation effort.

We seems to be having issues with chromatic building.

Resolve chromatic issue:

```
✖ Failed to extract stories from your Storybook
This is usually a problem with your published Storybook, not with Chromatic.

Build and open your Storybook locally and check the browser console for errors.
Visit your published Storybook at https://x.chromatic.com/
The following error was encountered while running your Storybook:

Error: page.evaluate: TypeError: Cannot destructure property 'story' of 'storyObject' as it is undefined.
    at normalizeStory (https://x.capture.chromatic.com/sb-preview/runtime.js:38:260)
    at https://x.capture.chromatic.com/sb-preview/runtime.js:41:400
    at Array.forEach (<anonymous>)
    at processCSFFile (https://x.capture.chromatic.com/sb-preview/runtime.js:41:354)
    at StoryStore.memoizerific [as processCSFFileWithCache] (https://x.capture.chromatic.com/sb-preview/runtime.js:1:4751)
    at https://x.capture.chromatic.com/sb-preview/runtime.js:47:8451
```

**Comment by TomRadford** [comment]
Note that this issue seems to be documented here:
https://github.com/chromaui/chromatic-cli/issues/703

**Comment by TomRadford** [comment]
The reason I thought the issue was related to chromatic cli is that I didn't *seem* to get any issues when running storybook locally.
Yesterday, when tinkering around a little more, I realised that chromatic's Capture Cloud seems to be generating story snapshots in sequential order. This means that some code that mutates my Svelte store as follows  `$page = { url: { pathname: '/settings/users' } };` was causing code in a later story to break `$page.param.id = '123'` (since param property would not exist).

Further, there were some missing imports in stories that I missed. 

Closing this.
